src/sentry/search/snuba/executors.py

```diff
@@ -1190,62 +1192,16 @@ def get_basic_event_snuba_condition(
         self,
         search_filter: SearchFilter,
         joined_entity: Entity,
-        organization_id: int,
-        project_ids: Sequence[int],
-        environments: Sequence[str],
+        snuba_params: ParamsType,
     ) -> Condition:
         """
         Returns the basic lookup for a search filter.
         """
-        # note this might hit postgres to do queries on releases
-        raw_conditions, projects_to_filter, group_ids = format_search_filter(
-            search_filter,
-            params={
-                "organization_id": organization_id,
-                "project_id": project_ids,
-                "environment": environments,
-            },
+        dataset = Dataset.Events if joined_entity.alias == "e" else Dataset.IssuePlatform
```

Comment:
Wondering if this should filter on `name` rather than `alias`? Either way, it might be better to use constants here